### ISSUE TITLE
'use strict' statement should not always require semicolon

### ISSUE THREAD

**Issue by JacobDanton** [palantir/tslint#351]
If I write something like:

```ts
export function test(): string {
    'use strict'
    return 'This is a test'
}
```

I get the lint error `missing 'use strict'`. However, adding a semicolon after that line makes it pass. Is there a reason for this behavior? I have 'semicolon' set to false in my config, so shouldn't that be taken into account for 'use-strict' checks?
